nixos/modules/config/shells-environment.nix

```
@@ -115,6 +115,11 @@ in
         aliases are added to all users' shells.
         Aliases mapped to `null` are ignored.
       '';
+      default = {
+        ls = "ls --color=tty";
+        ll = "ls -l";
+        l  = "ls -alh";
+      };
```

Your change would in fact change the default. I'm not the one to make the call on defaults, and `git blame` shows they have been empty in the past 10 years.

If you want it to show up as an empty attrset on search.nixos.org, make it that.

```suggestion
      default = { };
```